### ISSUE TITLE
add paypal donation

### ISSUE THREAD

**Issue by lfkdev** [totakaro/nhos-llserver#6]
I love this project and want to donate! But I want to hodl my Bitcoin so please add a way to donate over paypal, thx!

**Comment by Nomad965** [comment]
I agree on the same it's a great project, suggest maybe use https://www.patreon.com/

**Comment by totakaro** [comment]
Hi @lfkdev @Nomad965 Thank you for your suggestions, I have no paypal or patreon yet, my friend is helping me to create one, so I will update my donation page when my account is ready.

Happy mining! 🚀

**Comment by totakaro** [comment]
I just added a patreon link to my donation page: https://totakaro.github.io/donate

Thank you very much guys! 😄 👍

**Comment by totakaro** [comment]
lol My patreon account just got blocked, no idea why, so I will be searching for another options 😅
